web/massastation/src/pages/store/MyPlugin.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+
+import {
+  Button,
+  Certificate,
+  MassaWallet,
+  Plugin,
+} from '@massalabs/react-ui-kit';
+import { FiArrowUpRight, FiRefreshCcw, FiTrash2 } from 'react-icons/fi';
+import { IMassaPlugin } from './MyStation';
+
+enum PluginStatus {
+  Up = 'Up',
+  Down = 'Down',
```

Comment:
on/off since it's what is set up on the designs

Reply:
this is what's returned by the back
                  enum: ["Starting", "Up", "Down", "ShuttingDown", "Crashed"]

Reply:
changed to ON/OFF